src/proto/nc_redis.c

```diff
@@ -2695,7 +2704,11 @@ redis_reply(struct msg *r)
     switch (r->type) {
     case MSG_REQ_REDIS_PING:
         return msg_append(response, rsp_pong.data, rsp_pong.len);
-
+    case MSG_REQ_REDIS_SELECT:
```

Comment:
Alternately or additionally, it may be safe to allow forwarding this entirely if the db number is equivalent to the one in the config file for redis_db (`pool->redis_db`)